storage/storagepb/kv.proto

```diff
@@ -32,5 +32,7 @@ message Event {
   // a delete/expire event contains the previous
   // key-value
   KeyValue kv = 2;
+  // watchID is the identification of watching
```

Comment:
this is not quite correct.

watchID is the ID of watching this event is sent to.